DOCS-CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+
+To make changes to existing content, you can simply modify your local copy with a text editor, save, do your `git commit` and `git push`, and finally submit a PR with your changes via your clone or through your fork.
+
+If you want to create new content, there are some considerations to make regarding [Front Matter](https://gohugo.io/content-management/front-matter/) metadata, particularly around the `weight` attribute, which determines the order that the content gets displayed relative to the folder it's in within the left sidebar menu.
+
+If you want to create a new file, you could either simply create the .md file, give it a name and the appropriate front matter, or use the [hugo cli](https://gohugo.io/commands/):
+
+`hugo new docs/path/to/yourfile.md`
+
+This will create the file with some default front matter to get started, but you'll need to fill in the author, linkName and date values, as well as assign it an appropriate weight.
+
+For reference, just have a look at any of the other markdown content within /docs/ and you can see the front matter at the very top.
+
+
+
+#### Local Site Build Prerequisites
+
+If you want to build and test the site locally, you'll need the following:
+
+- Hugo installed locally (min version 0.75, but always try to use the latest version)
+- Node.js and the following node modules:
+  - autoprefixer
+  - postcss-cli
+  - postcss
+- A clone or fork of this repo with `photon-hugo` as the active branch.
+
+Hugo and this site can be built on x86 or ARM64 CPUs thanks to Hugo having being compiled to run on ARM64. (i.e. it can be built on an M1 Mac or Raspberry Pi without emulation. yay!)
+
+#### Installing Dependencies
+
+- Firstly, you'll need to have Hugo installed. [Here is a link to Hugo's documentation](https://gohugo.io/getting-started/installing/) for more information on how to do that for your operating system.
+
+- Install [Node.JS](https://nodejs.org/en/download/package-manager/) for your platform so you can use `npm`
+
+- Install PostCSS dependencies via `npm`:
+  - `cd` to the directory where your clone is located
+  - run: `npm install -D autoprefixer postcss postcss-cli`
+
+Once those are installed, you should be able to run `hugo` to build the site locally or `hugo server` to run the site in a live-reload local environment.
+
+#### Building and Reviewing your Changes Locally
+
+You can simply make changes to the Markdown content without needing to build the entire site locally. 
+
+However, if you want to review the changes locally in what it's final form should look like, you would run the `hugo server` command from the root of the repo.
+
+`hugo server` will build the site in real-time and present it to you at //localhost:1313.  Any changes that happen to markdown or design content will automatically be rebuild and loaded into the site when it detects that files have changed.
+
```

Comment:
http missing? //localhost:1313 -> http://localhost:1313. 
Extra space after period ```.  Any changes that happen```

Reply:
yah it'd be fine to use http:, I was just referencing what `hugo` output looks like (which says //localhost:1313)